cmd/podman/containers/create.go

```diff
@@ -114,6 +114,11 @@ func commonFlags(cmd *cobra.Command) error {
 	if cmd.Flags().Changed("image-volume") {
 		cliVals.ImageVolume = cmd.Flag("image-volume").Value.String()
 	}
+
+	if cmd.Flags().Changed("read-write-tmpfs") && !cliVals.ReadOnly {
+		return errors.New("--read-write-tmpfs is only used if --read-only=true")
```

Comment:
I think this would be clearer as "...is only **valid** if", or "...is only meaningful when..."